Mathlib/Topology/Semicontinuous.lean

```diff
@@ -5,6 +5,8 @@ Authors: Sébastien Gouëzel
 -/
 import Mathlib.Algebra.IndicatorFunction
 import Mathlib.Topology.ContinuousOn
+import Mathlib.Order.LiminfLimsup
+import Mathlib.Topology.Sequences
```

Comment:
```suggestion
import Mathlib.Topology.Sequences
```
The other import is already transitively import.

Note to reviewers: this doesn't import any new files transitively, so it's safe.